src/bci_build/package/cosign/README.md.j2

```diff
@@ -0,0 +1,51 @@
+# {{ image.title }}
+{% include 'badges.j2' %}
+
+## Description
+Cosign aims to make signatures invisible infrastructure.
+
+Cosign supports:
+
+* "Keyless signing" with the Sigstore public good Fulcio certificate authority and Rekor transparency log (default)
+* Hardware and KMS signing
+* Signing with a cosign generated encrypted private/public keypair
+* Container Signing, Verification and Storage in an OCI registry.
+* Bring-your-own PKI
+
+
+## Usage
+
+### ### Verify a container
+
+To verify the image, you'll need to pass in the expected certificate subject
+and certificate issuer via the `--certificate-identity` and
+`--certificate-oidc-issuer` flags:
+
+```
+podman run {{ image.pretty_reference }} verify $IMAGE --certificate-identity=$IDENTITY --certificate-oidc-issuer=$OIDC_ISSUER
+```
+
+You can also pass in a regex for the certificate identity and issuer flags,
+`--certificate-identity-regexp` and `--certificate-oidc-issuer-regexp`.
+
+### Verify a container against a public key
```

Comment:
```suggestion
### Verify a container image against a public key
```

Reply:
why the image addition? this is literally the copy of the upstream readme...

Reply:
Because you're verifying a container image, not a container. And upstream is not necessarily right all the time.